webserver/__init__.py

```diff
@@ -10,6 +10,10 @@ def create_postgres():
     from postgres_connection import init_postgres_connection
     return init_postgres_connection(current_app.config['SQLALCHEMY_DATABASE_URI'])
 
+def create_redis():
```

Comment:
Before i update it, i just wanted to clarify one thing.
The code creates 2 connection as well for the postgres database.
1. To display the listens, which is create at every request. (https://github.com/metabrainz/listenbrainz-server/blob/master/webserver/views/user.py#L42)
2. A persistent connection from `redis_consumer` that inserts the listens.(https://github.com/metabrainz/listenbrainz-server/blob/master/redis-consumer/redis-consumer.py#L31)

So i followed the similar logic,
1. A temporary redis connection to fetch `playing_now`
2. A persistent connection from `redis_consumer` that pushes listens from the redis queue to postgres.

Was this done intentionally?